app/hooks/use-nostr-profile.ts

```diff
@@ -1,10 +1,17 @@
 import { useState, useEffect } from "react"
 import * as Keychain from "react-native-keychain"
-import { nip19, generateSecretKey } from "nostr-tools"
+import { nip19, generateSecretKey, getPublicKey, SimplePool } from "nostr-tools"
 
 const useNostrProfile = () => {
   const KEYCHAIN_NOSTRCREDS_KEY = "nostr_creds_key"
   const [nostrSecretKey, setNostrSecretKey] = useState<string>("")
+  const relays = [
+    "wss://relay.damus.io",
+    "wss://relay.primal.net",
+    "wss://nostr.pleb.network",
```

Comment:
you can remove this one "nostr.pleb.network" - its offline and we will be changing the domain name